post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-ibm-z.adoc

```diff
@@ -1,8 +1,8 @@
 :_mod-docs-content-type: ASSEMBLY
 :context: creating-multi-arch-compute-nodes-ibm-z
-include::_attributes/common-attributes.adoc[]
 [id="creating-multi-arch-compute-nodes-ibm-z"]
-= Creating a cluster with multi-architecture compute machines on {ibm-z-title} and {ibm-linuxone-title} with z/VM
+= Creating a cluster with multi-architecture compute machines on IBM Z and IBM LinuxONE with z/VM
```

Comment:
Is there a reason you're removing the attributes from this heading? If there's no specific reason, the attributes should be used.

Reply:
I noticed that these attributes were not resolved properly in the preview. The issue persisted even after multiple retries, so I replaced the attributes with their actual values.